# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/gradient/reftest.list
# Commit: 6fc108de5c8f
# Full Hash: 6fc108de5c8fd527d0f5b6ae7b91604cc22e7d62
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2021-05-03 15:32:34
# Description:
#   Bug 1707744 - Avoid far gradient endpoints causing large gradient segments. r=gfx-reviewers,lsalzman
#   
#   Large segment bounds trip an assertion when casting coordinates to integers. Clipping early also reduces the amount of cached pixels.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113819
# ==============================================================================

diff -r 5d436e803f68 -r 6fc108de5c8f gfx/wr/wrench/reftests/gradient/reftest.list
--- a/gfx/wr/wrench/reftests/gradient/reftest.list	Mon May 03 09:12:42 2021 +0000
+++ b/gfx/wr/wrench/reftests/gradient/reftest.list	Mon May 03 09:12:42 2021 +0000
@@ -115,3 +115,6 @@
 fuzzy(2,23000) == linear-large.yaml linear-large-ref.yaml
 == conic-large.yaml conic-large-ref.yaml
 fuzzy(1,7000) == radial-large.yaml radial-large-ref.png
+
+# crash tests
+== linear-far-endpoints.yaml linear-far-endpoints.yaml
